### ISSUE TITLE
TypeError: path.includes is not a function

### ISSUE THREAD

**Issue by nitedani** [brillout/vite-plugin-ssr#786]
### Description

Error in `isImportBuildOutsideOfCwd` - this is because I bundle the server output(dist/server) of vite build with webpack into a single file(main.mjs). This was working before, but now it's not.

```ts
function isImportBuildOutsideOfCwd(paths) {
---
    console.log({importBuildFileName_1, paths});
    if (isWebpackResolve(importBuildFileName_1.importBuildFileName)) // false
        return null;
   ...
}
```
```
/***/ 58571:
/***/ ((__unused_webpack_module, exports) => {


Object.defineProperty(exports, "__esModule", ({ value: true }));
exports.importBuildFileName = void 0;
exports.importBuildFileName = 'importBuild.cjs';


/***/ }),
```

### Error Message + Error Stack

```shell
TypeError: path.includes is not a function
    at assertPosixPath (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:689:40)
    at toPosixPath (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:677:9)
    at isImportBuildOutsideOfCwd (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:374:51)
    at loadServerBuild (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:336:13)
    at loadImportBuild (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:53299:53)
    at initGlobalContext (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:53202:74)
    at renderPage (file:///home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/main.mjs:55176:53)
```
```
{
  importBuildFileName_1: { importBuildFileName: 'importBuild.cjs' },
  paths: {
    autoImporterFilePathOriginal: '/home/dani/projects/almafa/livekit-test/node_modules/.pnpm/@brillout+vite-plugin-import-build@0.2.14/node_modules/@brillout/vite-plugin-import-build/dist/autoImporter.js',
    autoImporterFileDirActual: '/home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist',
    importBuildFilePathRelative: '../../../../../../../apps/nodejs/dashboard/dist/server/importBuild.cjs',
    importBuildFilePathOriginal: '/home/dani/projects/almafa/livekit-test/apps/nodejs/dashboard/dist/server/importBuild.cjs',
    importBuildFilePathResolved: [Function: importBuildFilePathResolved]
  }
}
```
```

**Comment by brillout** [comment]
Can you provide a reproduction? Ideally a minimal one starting of `/examples/react/`, so that we can add it to VPS's CI.

**Comment by nitedani** [comment]
https://stackblitz.com/github/nitedani/vps-webpack-987654?file=package.json

**Comment by brillout** [comment]
Is vavite needed for the reproduction?

**Comment by nitedani** [comment]
It's not needed, removed it.

**Comment by brillout** [comment]
👌. I'll have a look at it later today. Let me know if it's urgent.

**Comment by brillout** [comment]
Fix released at `@brillout/vite-plugin-import-build@0.2.15`. Let me know if you run into another issue.

**Comment by brillout** [comment]
webpack is now part of the CI: https://github.com/brillout/vite-plugin-ssr/tree/main/examples/webpack-v1.
